web/src/app/admin/connectors/confluence/page.tsx

```diff
@@ -207,13 +210,17 @@ const Main = () => {
           <p className="text-sm mb-4">
             Specify any link to a Confluence page below and click
             &quot;Index&quot; to Index. Based on the provided link, we will
-            index the ENTIRE SPACE, not just the specified page. For example,
-            entering{" "}
+            index either the entire page and its subpages OR the entire space.
+            For example, entering{" "}
             <i>
               https://danswer.atlassian.net/wiki/spaces/Engineering/overview
             </i>{" "}
             and clicking the Index button will index the whole{" "}
-            <i>Engineering</i> Confluence space.
+            <i>Engineering</i> Confluence space, but entering
+            https://pablosfsanchez.atlassian.net/wiki/spaces/Engineering/pages/164331/example+page
+            will index that page&apos;s children (and optionally, its children).
```

Comment:
I think `optionally, its children` should be something like `optionally, itself`?